lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -787,9 +787,10 @@ def get_est_cost_using_known_size(self, uri, size):
 
         cost = self._get_est_cost_from_stream_size(size)
 
-        resolved = yield self.session.wallet.resolve_uri(uri)
-        if 'claim' in resolved:
-            claim = ClaimDict.load_dict(resolved['claim']['value'])
+        resolved = yield self.session.wallet.resolve(True, 0, 10, uri)
```

Comment:
Please use named params here in `resolve(True, 0, 10, uri)`. `True`, `0`, and `10` are meaningless when you read this code. Or if you rearrange params as in the above comment, you only have to pass in the uris.